flatbuffers/MarketDataSnapshot.fbs

```diff
@@ -56,4 +56,8 @@ table MarketDataSnapshot {
     bid: [MDEntry];
     /// Offer market data entries.
     offer: [MDEntry];
+    /// Dodd-Frank mid price.
+    mid_price: double;
+    /// Swaps only, far leg Dodd-Frank mid price.
+    far_mid_price: double;
```

Comment:
Don't think we need this at the moment, no swaps over ESP.

Reply:
Fair, was thinking it is OK to add as we will use it eventually, but I see all other far leg related fields are missing too. Removed